ci/kokoro/ubuntu/build.sh

```diff
@@ -51,15 +51,10 @@ echo "https://source.cloud.google.com/results/invocations/${INVOCATION_ID}"
 echo "================================================================"
 echo ${INVOCATION_ID} >> "${KOKORO_ARTIFACTS_DIR}/bazel_invocation_ids"
 
-bazel --bazelrc=kokoro-bazelrc test \
+bazel test \
     --test_output=errors \
     --verbose_failures=true \
     --keep_going \
```

Comment:
Not sure why these flags aren't needed anymore, but I'll trust you on that :P

Reply:
Thanks. For the record, those were an attempt to use the fancy UI to see the test results, it never worked, and it is just making the runs slow.